src/mongoose_privacy.erl

```diff
@@ -63,14 +65,8 @@ privacy_check_packet(Acc0, Server, User, PrivacyList, From, To, Dir) ->
     Key = {cached_check, Server, User, From, To, Name, Type, Dir},
     case mongoose_acc:get(privacy, Key, undefined, Acc) of
         undefined ->
-            Acc1 = ejabberd_hooks:run_fold(privacy_check_packet,
-                                           Server,
-                                           mongoose_acc:set(hook, result, allow, Acc),
```

Comment:
is it safe to remove this `mongoose_acc:set/4 call`?

Reply:
It is not removed. It's moved to the `mongoose_hooks:privacy_check_packet` function here: https://github.com/esl/MongooseIM/pull/2642/files#diff-95c7ec185b60cdba90f25da513edbc75R195